src/renderer.ts

```diff
@@ -38,27 +43,31 @@ export class Renderer {
   private render() {
     const { t, factory } = this;
 
-    const score = t.const('score', () => factory.EasyScore());
     t.newline();
 
-    let system = t.let<VF.System | undefined>('system', () => undefined);
     let timeSignature = t.let('timeSignature', () => '');
-    let clefs = t.let('clefs', () => new Array<string>());
     let clef = t.let('clef', () => '');
-    let curClefs: string[] = [];
-    let voices = t.let('voices', () => new Array<VF.Voice>());
-    let note = t.let<VF.StemmableNote | undefined>('note', () => undefined);
-    let notes = t.let('notes', () => new Array<VF.StemmableNote>());
+    let stave: VF.Stave | undefined = t.let('stave', () => undefined);
+    const staves: Map<number, VF.Stave> = t.let('staves', () => new Map<number, VF.Stave>());
```

Comment:
nit: Here and below, use `t.const` instead so it prints correctly.

Reply:
Done